GUI.cpp

```diff
@@ -359,7 +359,7 @@ namespace
     {
         auto& ssConfig = stonesenseState.ssConfig;
 
-        ALLEGRO_COLOR color = ssConfig.config.colors.getDfColor(report->color, ssConfig.config.useDfColors);
+        ALLEGRO_COLOR color = ssConfig.config.colors.getDfColor(report->color, true, ssConfig.config.useDfColors);
```

Comment:
`bright` is `int` in `getDfColor`. while i realize there is an implicit conversion from `bool` to `int`, i don't approve of leaning into it, and would prefer that _either_ `getDfColor` be written to take `bright` as a bool, or that this call pass a proper `int` instead of relying on the implicit `int` to `bool` conversion